EMOlater/backend/views.py

```diff
@@ -1,3 +1,42 @@
 from django.shortcuts import render
 
 # Create your views here.
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import viewsets, status
+
+from .models import User, SurveyRepsponse
+from .serializers import UserSerializer, SurveyRepsponseSerializer
+
+class UserViewSet(viewsets.ModelViewSet):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+class SurveyRepsponseViewSet(viewsets.ModelViewSet):
+    queryset = SurveyRepsponse.objects.all()
+    serializer_class = SurveyRepsponseSerializer
+
+@api_view(['GET', 'POST'])
+def user(request):
+    if request.method == 'POST':
```

Comment:
since the POST and GET paths are entirely independent, maybe it would make sense to have them be separate functions

Reply:
I wanted to separate them for sure, but I'm not familiar how can I do this with the framework.